dipy/segment/mask.py

```diff
@@ -8,7 +8,7 @@
 
 from scipy.ndimage.filters import median_filter
 try:
-    from skimage.filter import threshold_otsu as otsu
+    from skimage.filters import threshold_otsu as otsu
```

Comment:
Cool. That answers my question.

On Mon, Sep 21, 2015 at 12:34 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In dipy/segment/mask.py
> <https://github.com/nipy/dipy/pull/695#discussion_r40014939>:
>
> > @@ -8,7 +8,7 @@
> >
> >  from scipy.ndimage.filters import median_filter
> >  try:
> > -    from skimage.filter import threshold_otsu as otsu
> > +    from skimage.filters import threshold_otsu as otsu
>
> I did this change. There is a warning that skimage has deprecated
> skimage.filter and that now we should use skimage.filters.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/695/files#r40014939>.
>